### DISCUSSION TITLE
v2.7.5 LTS release date

### DISCUSSION THREAD

**Discussion by seonghoon-ahn** [zephyrproject-rtos/zephyr#57766] [Q&A] [answered]
Hi all,

I expected Zephyr v2.7.5 to be released in April, but it was not.
When will it be released?

Thank you.

**Comment by kartben** [comment, marked as answer]
Please see https://lists.zephyrproject.org/g/devel/message/8976 -- **release is planned for next week, on May 26**.
Also please note @cfriedt's comment on the removal of TF-M integration in this release.

**Reply by cfriedt** [reply]
Thanks - yeah, I did announce delays via the devel mailing list a couple of times, but since it was a regular topic at the TSC meeting until now, it seemed a bit redundant to keep doing so until there was resolution from the TSC.
